docs/Rules.md

```diff
@@ -1966,3 +1964,41 @@ The only catch is that the protocol needs to be setup in the rule. Most likely t
 
 ------------------------------------------------------------------------------
 
+### RfRaw command duplicator
+
+Sonoff RF bridge with Tasmota firmware is able to send out the radio-frequency commands very quickly. If some of your covers 'miss' the commands occassionally (you can see that from the fact that the state shown in the HA system does not correspond to reality), it may be that those cover motors do not pick up the codes immediately when they are sent.
```

Comment:
This should probably be rewritten not just specific to curtain motors.

Reply:
or mentioned "as an example"

Reply:
The whole cookbook is a collection of examples actually, no need to mention again. It is not specific to curtain motors, but this happens most often with those, so this description was written with google search in mind, to find here the solution.